README.md

```diff
@@ -37,8 +37,8 @@ pnpm add react-youtube
   videoId={string}                  // defaults -> ''
   id={string}                       // defaults -> ''
   className={string}                // defaults -> ''
-  containerClassName={string}       // defaults -> ''
-  containerStyle={object}           // defaults -> {}
+  iframeClassName={string}           // defaults -> ''
+  style={object}           // defaults -> {}
```

Comment:
Please make sure the comments are horizontally aligned with each other.

Reply:
Fixed in 9ca943010447808bbc57b142d51de3d96cb0ecda.